src/controller/CHIPDeviceControllerSystemState.h

```diff
@@ -127,9 +130,9 @@ class DeviceControllerSystemState
         mUnsolicitedStatusHandler(params.unsolicitedStatusHandler), mExchangeMgr(params.exchangeMgr),
         mMessageCounterManager(params.messageCounterManager), mFabrics(params.fabricTable), mCASEServer(params.caseServer),
         mCASESessionManager(params.caseSessionManager), mSessionSetupPool(params.sessionSetupPool),
-        mCASEClientPool(params.caseClientPool), mGroupDataProvider(params.groupDataProvider),
-        mSessionKeystore(params.sessionKeystore), mFabricTableDelegate(params.fabricTableDelegate),
-        mSessionResumptionStorage(std::move(params.sessionResumptionStorage))
+        mCASEClientPool(params.caseClientPool), mGroupDataProvider(params.groupDataProvider), mTimerDelegate(params.timerDelegate),
```

Comment:
Now that we are always providing these things, shouldn't IsInitialized() check for them like the previous version?